composer.json

```diff
@@ -25,10 +25,7 @@
     "autoload": {
         "psr-4": {
             "Spatie\\TwitterStreamingApi\\": "src"
-        },
-        "classmap": [
```

Comment:
Yeah, it's a pain. But I prefer that to a user having to manually pull in another package when installing this one.

I'll dig around to found out how this can be improved.